### DISCUSSION TITLE
C# windows forms gravity error

### DISCUSSION THREAD

**Discussion by ghost** [community/community#68341] [Programming Help]
### Body

Hi, I was making a game in C# and ran into an error the link is here:  https://github.com/animeReview/SleepingDragonsGame/blob/main/src/Form1.cs https://github.com/animeReview/SleepingDragonsGame/blob/main/src/Form1.cs

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebubekirgungor** [comment]
Line 14: public int gravity = -10;  // this is negative
You can try turning it to positive.
